sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -315,7 +315,7 @@ def decode_from_stream(self, stream, nested):
       vlen = stream.read_var_int64()
       vlist = [self.decode_from_stream(stream, True) for _ in range(vlen)]
       if t == LIST_TYPE:
-        return vlist
+        return list(vlist)
```

Comment:
Why this change?

Reply:
So explicitly converts the generator into a list for Py3.